package.json

```diff
@@ -37,27 +37,26 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "git-branch-is": "^4.0.0",
     "husky": "^4.3.7",
-    "node-sass": "^4.14.1",
     "npm-run-all": "^4.1.5",
     "npm-version-suffix": "^0.2.0",
     "ora": "^5.4.0",
     "postcss": "^8.2.8",
     "react-scripts": "^4.0.3",
-    "react-select": "^4.3.0",
+    "react-select": "^4.3.1",
     "rollup": "^2.43.1",
     "rollup-plugin-copy": "^3.3.0",
     "rollup-plugin-delete": "^2.0.0",
     "rollup-plugin-dts": "^2.0.1",
     "rollup-plugin-node-externals": "^2.2.0",
     "rollup-plugin-replace": "^2.2.0",
-    "rollup-plugin-scss": "^2.6.1",
+    "rollup-plugin-scss": "^3.0.0",
     "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-typescript2": "^0.30.0",
+    "rollup-plugin-typescript2": "^0.31.0",
     "typescript": "^4.2.3"
   },
   "peerDependencies": {
-    "react": "^16.13.1 || ^17.0.0",
-    "react-dom": "^16.13.1 || ^17.0.0"
+    "react": "^18.2.0",
```

Comment:
Do I understand correctly that the new version is only compatible with React 18? Or is React 17 still supported?